packages/react-client/src/ReactFlightClient.js

```diff
@@ -689,6 +694,18 @@ function parseModelString(
         const metadata = getOutlinedModel(response, id);
         return createServerReferenceProxy(response, metadata);
       }
+      case 'T': {
+        // Temporary Reference
+        const id = parseInt(value.slice(2), 16);
+        const temporaryReferences = response._tempRefs;
+        if (temporaryReferences == null) {
```

Comment:
Saw that the type is `void | TemporaryReferenceSet` and in the other place we're checking with `=== undefined`. Is the `== null` expected?

Reply:
Yea, I just do it since it's a public API and a user might still pass `null` incorrectly but really we should probably be more strongly opinionated about not doing that.